Source/WebCore/editing/cocoa/HTMLConverter.mm

```diff
@@ -2412,7 +2412,18 @@ static String preferredFilenameForElement(const HTMLImageElement& element)
     return wrapper;
 }
 
-static RetainPtr<NSAttributedString> attributedStringWithAttachmentForElement(const HTMLImageElement& element)
+template<typename Element>
+static RetainPtr<NSAttributedString> attributedStringWithAttachmentForElement(const Element& element)
```

Comment:
I'd like this method name to be a bit more specific / refer to file wrappers. As it stands it's very easy for new callers to use this for `HTMLImageElement`, without calling into `attributedStringWithAttachmentForImageElement`.

Reply:
refactored